### PR TITLE
Ex 49 meeting based tasks july 12 minor

### PR DESCRIPTION
**Pull request by KellyInnovation**

### CODE REVIEW

File: lfucg-exactions/client/js/components/Navbar.js [hunk 1]
```diff
@@ -24,12 +24,22 @@ class Navbar extends React.Component {
...
 
+        const windowWidth = window.innerWidth;
+
         return (
             <div className="react-navbar">
                 <header>
                     <div className="container">
                         <div className="navbar-header">
-                            <Link to="dashboard/" className="navbar-brand" aria-label="Exactions Home" role="link" />
+                            {windowWidth <= 767 ?
```
Comment (jarane2) [inline lfucg-exactions/client/js/components/Navbar.js]: Fine as is, but is there an scss way to handle this instead?

File: lfucg-exactions/client/sass/_navbar.scss [hunk 2]
```diff
@@ -12,10 +12,7 @@ header {
         height: 70px;
     }
     .navbar-brand {
-        background-image: url('../../static/Lexington_Web_Color_Horse_Only.png');
-        background-repeat: no-repeat;
-        background-size: 100%; 
-        width: 50px;
+        /*width: 50px;*/
```
Comment (jarane2) [inline lfucg-exactions/client/sass/_navbar.scss]: related to previous comment, it looks like this used to be a scss solution.  Was this causing problems?

File: lfucg-exactions/server/server/settings/base.py [hunk 3]
```diff
@@ -234,7 +234,7 @@ def custom_show_toolbar(request):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'exactions.db',
+        'NAME': 'exactions',
```
Comment (jarane2) [inline lfucg-exactions/server/server/settings/base.py]: Was this recommended by Chris or Wei-Keat?  Just curious.  Also just need to make sure this file is gitignored

File: lfucg-exactions/client/js/components/Navbar.js (hunk 1 above)
Comment (KellyInnovation) [inline lfucg-exactions/client/js/components/Navbar.js]: SCSS was not able to concatenate.

**Pull request merged**